ivadomed/main.py

```diff
@@ -388,9 +398,8 @@ def cmd_train(context):
 
             num_steps += 1
 
-            # Only write sample at the first step
             if i == 0:
-                utils.save_tensorboard_img(writer, epoch, "Train", input_samples, gt_samples, preds, unet_3D)
+                imed_utils.save_tensorboard_img(writer, epoch, "Train", input_samples, gt_samples, preds, unet_3D)
```

Comment:
I should have communicated with you about this changes :/ duplication. Not a worry tho